<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPwdWidget</class>
 <widget class="QWidget" name="SetPwdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>454</width>
    <height>262</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>454</width>
    <height>262</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SetPwdWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#SetPwdWidget{background-color: rgb(255, 255, 255);border-radius:8px;}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>19</y>
     <width>91</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>设置新密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>5</y>
     <width>32</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>28</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/close.png);}
QPushButton:hover,pressed{border-image: url(:/EntCenterWidget/image/close_sel.png);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNewPwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>268</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>268</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>268</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入新登录密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>191</y>
     <width>72</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:disabled{ background-color:rgba(200,200,200,1);border-radius:4px;};</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComfirmPwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>268</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>268</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>268</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请再次输入新登录密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>191</y>
     <width>72</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#FFFFFF;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#333333;border:1px solid #DEDEDE;}
QPushButton:hover,pressed{background-color:#F4F4F4;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#333333;}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>125</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>&lt;font color = red&gt;*&lt;/font&gt;&lt;font color =#333333&gt;确认密码&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>66</y>
     <width>56</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>&lt;font color = red&gt;*&lt;/font&gt;&lt;font color =#333333&gt;新密码&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>211</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>请输入8～20个字母、数字或特殊字符</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>211</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>请输入8～20个字母、数字或特殊字符</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections/>
</ui>
